README.md

```diff
@@ -1,2 +1,102 @@
-# ephemeral-environment-action
-Github action for creating and deleting an ephemeral environment on molnett
+# Molnett Ephemeral Environments
+
+This Github action makes it easier to use the ephemeral environments feature of molnett. The feature is intended for creating a deployment for each pull request.
+
+## Usage
+
+### Minimal example
+
+Pull request build:
+
+```yaml
+      - name: Deploy Ephemeral Environment
+        id: deploy
+        uses: molnett/ephemeral-environment-action@v1
+        with:
+          action: deploy
+          manifest-path: molnett.yaml
+```
+
+Main branch build:
+
+```yaml
+    - name: Delete Ephemeral Environment
+      id: delete
+      uses: molnett/ephemeral-environment-action@v1
+      with:
+        action: delete
+        manifest-path: molnett.yaml
+```
+
+### Full example with setup, docker build and pr comment
+
+```yaml
+name: "PR"
+
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - synchronize
+
+jobs:
+  create-ephemeral-env:
+    name: Ephemeral Environment
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+      contents: read
+    steps:
+      - uses: actions/checkout@v4
+      - name: Molnctl Setup
+        uses: molnett/setup-molnctl-action@v2
+        with:
+          api-token-client-id: ${{ secrets.MOLNETT_CLIENT_ID }}
+          api-token-client-secret: ${{ secrets.MOLNETT_CLIENT_SECRET }}
+          default-org: demo
+      - name: Build & Push Image to Molnett
+        run: |
+          molnctl auth docker
+          IMAGE_NAME=`molnctl svcs image-name -u molnett.yaml`
```

Comment:
Backticks are not recommended in Bash in general as it is only currently maintained for backwards compatibility. The general suggestion is to use `$()` for substitions.